Bottom: 368848ea60b78ec10bbf4d64aef14b841e84ebcc
Top:    d9336de566c7821bc92487033abe56ab1730359c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 12:26:11 +0100

Remove traces of LoadBackgroundImage() from the core of PCB


---

diff --git a/src/draw.h b/src/draw.h
index 484ae98..ef32ebb 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -70,8 +70,5 @@ void EraseElement (ElementTypePtr);
 void EraseElementPinsAndPads (ElementTypePtr);
 void EraseElementName (ElementTypePtr);
 void EraseObject (int, void *, void *);
-void LoadBackgroundImage (char *);
-
-/*GdkDrawable *draw_get_current_drawable(void);*/
 
 #endif
diff --git a/src/main.c b/src/main.c
index 2f4322f..5ce6e53 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1052,9 +1052,6 @@ main (int argc, char *argv[])
       LayerStringToLayerStack (Settings.InitialLayerStack);
     }
 
-  /*    FIX_ME
-     LoadBackgroundImage (Settings.BackgroundImage); */
-
   /* This must be called before any other atexit functions
    * are registered, as it configures an atexit function to
    * clean up and free various items of allocated memory,
